defaults.json

```diff
@@ -0,0 +1,113 @@
+{
+	"roles": {
+		"Administrator": {
+			"name": "employee.role.Administrator.name",
+			"description": "employee.role.Administrator.description",
+			"canManageOrganization": true,
+			"canManageEmployees": true,
+			"canManageRoles": true,
+			"canManageIntegrations": true,
+			"canManageProperties": true,
+			"canManageTickets": true,
+			"canManageContacts": true,
+			"canManageTicketComments": true
+		},
+		"Dispatcher": {
+			"name": "employee.role.Administrator.name",
+			"description": "employee.role.Administrator.description",
+			"canManageOrganization": false,
+			"canManageEmployees": false,
+			"canManageRoles": false,
+			"canManageIntegrations": false,
+			"canManageProperties": true,
+			"canManageTickets": true,
+			"canManageContacts": true,
+			"canManageTicketComments": true
+		},
+		"Manager": {
+			"name": "employee.role.Administrator.name",
+			"description": "employee.role.Administrator.description",
+			"canManageOrganization": false,
+			"canManageEmployees": false,
+			"canManageRoles": false,
+			"canManageIntegrations": false,
+			"canManageProperties": true,
+			"canManageTickets": true,
+			"canManageContacts": true,
+			"canManageTicketComments": true
+		},
+		"Foreman": {
+			"name": "employee.role.Administrator.name",
+			"description": "employee.role.Administrator.description",
+			"canManageOrganization": false,
+			"canManageEmployees": false,
+			"canManageRoles": false,
+			"canManageIntegrations": false,
+			"canManageProperties": false,
+			"canManageTickets": true,
+			"canManageContacts": false,
+			"canManageTicketComments": true
+		},
+		"Technician": {
+			"name": "employee.role.Administrator.name",
+			"description": "employee.role.Administrator.description",
+			"canManageOrganization": false,
+			"canManageEmployees": false,
+			"canManageRoles": false,
+			"canManageIntegrations": false,
+			"canManageProperties": false,
+			"canManageTickets": true,
+			"canManageContacts": false,
+			"canManageTicketComments": true
+		}
+	},
+	"statuses": {
+		"types": [
+			{
+				"type": "OPEN",
+				"name": "ticket.status.OPEN.name"
+			},
+			{
+				"type": "IN_PROGRESS",
+				"name": "ticket.status.IN_PROGRESS.name"
+			},
+			{
+				"type": "DECLINED",
+				"name": "ticket.status.DECLINED.name"
+			},
+			{
+				"type": "COMPLETED",
+				"name": "ticket.status.COMPLETED.name"
+			},
+			{
+				"type": "DEFERRED",
+				"name": "ticket.status.DEFERRED.name"
+			},
+			{
+				"type": "CLOSED",
+				"name": "ticket.status.CLOSED.name"
+			}
+		],
+		"transitions": {
```

Comment:
what is it?

Reply:
status transition based on locale

Reply:
statusTransions are not related to Localization. Why do we need to move it here?

Reply:
it's related on statusId, statusId relates to locale